djinni/jni/djinni_jni_main.cpp

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2021 cross-language-cpp
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "djinni_jni_main.hpp"
```

Comment:
This new files only purpose is to inclued the `JNI_OnLoad` implementation from `djinni_jni_main.hpp`.